docs/user/wireguard.md

```diff
@@ -1,31 +1,59 @@
 # Remote access
 
+When you visit a website (or the Brewblox UI), your browser sends multiple requests to the server.
+This includes requests for current block values and graph data, but also requests for block changes.
+For this to be safe, the server must be able to trust the client.
+In a local network, this is simple: everyone on the network is trusted.
+
+This is no longer true when connected to the internet.
+Here, every single request must include some form of authentication to confirm that the sender can be trusted.
+
```

Comment:
Very technical, I thought more about a practical introduction. For instance:
You might want to check the status of your brew, or manipulate the brew settings while away. For this to happen, we have to find a way to face the Brewblox UI to the internet. To have the Brewblox UI face the internet, and your brew settings to be secure requires password protection.